### PR TITLE
Add support for cc/bcc in SwiftMailerConsumer

### PR DESCRIPTION
**Pull request by gremo**
I am targeting this branch, because this is an enhancement.

## Changelog

```markdown
### Added
- Support for cc/bcc fields in `SwiftMailerConsumer`

```

### CODE REVIEW

File: Tests/Consumer/SwiftMailerConsumerTest.php [hunk 1]
```diff
@@ -72,6 +80,8 @@ public function testSendEmail()
         $mail->expects($this->once())->method('setFrom')->with($this->equalTo(array('from@mail.fr' => 'nameFrom')))->willReturnSelf();
         $mail->expects($this->once())->method('setTo')->with($this->equalTo(array('to1@mail.fr', 'to2@mail.fr' => 'nameTo2')))->willReturnSelf();
         $mail->expects($this->once())->method('setReplyTo')->with($this->equalTo(array('replyTo1@mail.fr', 'replyTo2@mail.fr' => 'nameReplyTo2')))->willReturnSelf();
+        $mail->expects($this->once())->method('setCc')->with($this->equalTo(array('cc1@mail.fr', 'cc2@mail.fr' => 'nameCc2')))->willReturnSelf();
+        $mail->expects($this->once())->method('setBcc')->with($this->equalTo(array('bcc1@mail.fr', 'bcc2@mail.fr' => 'nameBcc2')))->willReturnSelf();
```
Comment (greg0ire) [inline Tests/Consumer/SwiftMailerConsumerTest.php]: Please linebreak this

Comment (gremo) [inline Tests/Consumer/SwiftMailerConsumerTest.php]: The other lines aren't breaked, are you sure?

Comment (jordisala1991) [inline Tests/Consumer/SwiftMailerConsumerTest.php]: IMO we should linebreak them all. (If the line is more than 120 chars)

Comment (gremo) [inline Tests/Consumer/SwiftMailerConsumerTest.php]: Well, refactoring this class isn't the point of this PR. I'll stick with the original style.

Comment (core23) [inline Tests/Consumer/SwiftMailerConsumerTest.php]: Maybe break this like the following to gain some structure?

```
$mail->expects($this->once())
    ->method('setBcc')
    ->with($this->equalTo(array('bcc1@mail.fr', 'bcc2@mail.fr' => 'nameBcc2')))
    ->willReturnSelf();
```

Comment (greg0ire) [inline Tests/Consumer/SwiftMailerConsumerTest.php]: @gremo not the point of this PR indeed, but please linebreak anyway just your code sample.

**Comment by gremo** [comment]
Here we go people! Thanks for merging

**Comment by jordisala1991** [comment]
Thanks @gremo

**Pull request merged**